### PR TITLE
Add an information about CSRF vulnerability in Sylius

### PR DESCRIPTION
**Pull request by pamil**

### CODE REVIEW

**Comment by xabbuh** [comment]
Should there be entries for `sylius/admin-bundle` and `sylius/resource-bundle` too?

**Comment by pamil** [comment]
Yeah, I'll add AdminBundle as well, ResourceBundle isn't affected directly.

**Comment by pamil** [comment]
Done!

**Comment by Ocramius** [comment]
> Yeah, I'll add AdminBundle as well, ResourceBundle isn't affected directly.

Is the code of those directly affected, or just through the dependency?

File: sylius/admin-bundle/2018-07-09.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+branches:
+    1.0.x:
+        time: 2018-07-08 23:47:50
+        versions: ['>=1.0.0', '<1.0.17']
+    1.1.x:
+        time: 2018-07-09 00:09:14
+        versions: ['>=1.1.0', '<1.1.9']
+    1.2.x:
+        time: 2018-07-09 00:26:07
+        versions: ['>=1.2.0', '<1.2.2']
```
Comment (xabbuh) [inline sylius/admin-bundle/2018-07-09.yaml]: the newest release is 1.2.1

Comment (xabbuh) [reply sylius/admin-bundle/2018-07-09.yaml]: In fact, this applies to the other branches as well. It seems that subtree split releases are not always done when releases for the main repo are tagged.

Comment (pamil) [reply sylius/admin-bundle/2018-07-09.yaml]: All subtree split packages will be tagged in up to ~15 minutes.

**Comment by pamil** [comment]
@Ocramius AdminBundle is directly affected, ResourceBundle contains a new feature in a generic action, which validates a CSRF token (that action can also be used through REST API).

File: sylius/admin-bundle/2018-07-09.yaml (hunk 1 above)
Comment (pamil) [reply sylius/admin-bundle/2018-07-09.yaml]: Should be already tagged.

**Comment by fabpot** [comment]
Thank you @pamil

**Pull request merged**